client/src/App.css

```diff
@@ -52,24 +52,42 @@
   grid-row-end: 3;
 }
 
+#new-project-form,
 #overlay {
-  position: fixed;
   top: 0;
   bottom: 0;
   right: 0;
   left: 0;
+}
+
+#overlay {
+  position: fixed;
   background-color: var(--grey-90-a20);
```

Comment:
I feel like we should have a slightly darker overlay. Is there an `-a30` or `-a40`, by chance?

Reply:
Ooh! There's an [official recommendation](https://design.firefox.com/photon/patterns/overlay.html), even!